Dockerfile

```diff
@@ -26,14 +26,21 @@ RUN apt-get -y update \
 # Required by tonic
 RUN rustup component add rustfmt
 
+# Build UI
```

Comment:
Building the UI first and separately speeds things up when rebuilding the image locally.